docs/app/pages/components/components-sections/tables/column-sorting/column-sorting.component.ts

```diff
@@ -74,28 +75,36 @@ export class ComponentsColumnSortingComponent extends BaseDocumentationSection i
         active: chance.bool()
     }];
 
-    playground: IPlayground = {
-        files: {
-            'app.component.ts': this.snippets.raw.appTs,
-            'app.component.html': this.snippets.raw.appHtml
-        },
-        modules: [
-            {
-                imports: ['ColumnSortingModule', 'ColorServiceModule', 'SparkModule'],
-                library: '@ux-aspects/ux-aspects'
+    playground = () => {
+        return {
+            files: {
+                'app.component.ts': this.snippets.raw.appTs,
+                'app.component.html': this.snippets.raw.appHtml
             },
-            {
-                imports: ['A11yModule'],
-                library: '@angular/cdk/a11y'
-            }
-        ]
-    };
+            modules: [
+                {
+                    imports: ['ColumnSortingModule', 'ColorServiceModule', 'SparkModule'],
+                    library: '@ux-aspects/ux-aspects'
+                },
+                {
+                    imports: ['A11yModule'],
+                    library: '@angular/cdk/a11y'
+                }
+            ]
+        };
+    }
 
     sparkTrackColor = this._colorService.getColor('chart2').setAlpha(0.2).toRgba();
     sparkBarColor = this._colorService.getColor('chart2').toHex();
 
-    constructor(private _colorService: ColorService, private _announcer: LiveAnnouncer) {
+    constructor(
+        private _colorService: ColorService,
+        private _announcer: LiveAnnouncer,
+        @Inject(DOCUMENTATION_TOKEN) private _documentationType: DocumentationType) {
```

Comment:
I don't think this needs to be private because you aren't using it outside of the constructor.